course_info/templates/course_info/index.html

```diff
@@ -16,6 +16,7 @@
     <script src="{% static 'datatables-bootstrap3/BS3/assets/js/datatables.js' %}"></script>
     <script>
         window.globals.STATIC_URL = '{% settings_value "STATIC_URL" %}';
+        window.globals.COURSE_API_URL = '{% url "courseinstance-list" %}';
```

Comment:
Sorry, I just landed a change that pulls the term code dropdown data from the api as well...we need to update this to point to the api root, then append the resource name in the ajax call.